geoips/plugins/modules/output_formatters/geotiff_standard.py

```diff
@@ -90,15 +90,15 @@ def call(
             res_deg_x = (lons[-1][-1] - lons[0][0]) / width
             res_deg_y = (lats[0][0] - lats[-1][-1]) / height
 
-            minlat = area_def.area_extent_ll[1]
+            maxlat = area_def.area_extent_ll[-1]
             minlon = area_def.area_extent_ll[0]
 
             transform = Affine.translation(
-                minlon - res_deg_y / 2, minlat - res_deg_x / 2
-            ) * Affine.scale(res_deg_y, res_deg_x)
+                minlon - res_deg_x / 2, maxlat - res_deg_y / 2
```

Comment:
Maybe add a quick note as to what this is doing? (documentation!)

Reply:
I saw your comment below and will leave it as is for the time being.

Reply:
Well, I decided to add a note, for future reference and sanity.. Is that correct, @evrose54 ?

Reply:
Yup, your comment is correct @mindyls . I found another conversation on GitHub which pointed me in the right direction to fix this issue. https://github.com/rasterio/rasterio/issues/2565